### PR TITLE
fix(branding): Replace Elk branding with Mozilla

### PR DESCRIPTION
**Pull request by jpezninjo**
[SOCIALPLAT-95](https://mozilla-hub.atlassian.net/browse/SOCIALPLAT-95)

- [x] Replace the 7 assets under the `/public-dev` folder with Mozilla images
- [x] Replace the Elk logo on the home page with support for dark mode and mobile-view

Home
image
image
image
image

Toolbar
image

iOS
image

### CODE REVIEW

File: components/nav/NavTitle.vue [hunk 1]
```diff
@@ -18,20 +18,17 @@ router.afterEach(() => {
 </script>
 
 <template>
-  <div flex justify-between sticky top-0 bg-base z-1 py-4 native:py-7 data-tauri-drag-region>
+  <div flex sm-justify-center xl-justify-between sticky top-0 bg-base z-1 py-4 native:py-7 data-tauri-drag-region>
     <NuxtLink
       flex items-end gap-3
-      py2 px-5
+      py2 sm:px3
```
Comment (jpezninjo) [inline components/nav/NavTitle.vue:24]: spacing is conditional to account for the 2 different Home logos

File: components/nav/NavTitle.vue [hunk 2]
```diff
@@ -18,20 +18,17 @@ router.afterEach(() => {
 </script>
 
 <template>
-  <div flex justify-between sticky top-0 bg-base z-1 py-4 native:py-7 data-tauri-drag-region>
+  <div flex sm-justify-center xl-justify-between sticky top-0 bg-base z-1 py-4 native:py-7 data-tauri-drag-region>
```
Comment (jpezninjo) [inline components/nav/NavTitle.vue:21]: the mobile-view version of the Home logo will receive `justify-content: center` to match the other logos below in the column (the text next to each logo gets hidden in this state)

File: components/nav/NavLogo.vue [hunk 3]
```diff
@@ -1,54 +1,28 @@
...
-      <path
-        id="path26"
-        class="wood"
-        d="M 90.098,45.294 C 87.582,39.55 86.057,32.487 86.743,23.794 l 12.659,0.932 c -0.763,10.555 2.897,17.696 7.015,22.353 -5.338,-0.931 -10.447,-1.04 -16.319,-1.785 z m 80.069,-1.32 8.312,-9.702 c 21.58,19.094 8.159,46.415 8.159,46.415 l -11.819,-1.32 c -0.382,-6.24 -1.144,-17.836 -6.635,-24.371 3.584,1.84 6.635,3.865 9.99,6.908 0,-5.666 -1.754,-12.341 -8.007,-17.93 z"
-      />
-    </g>
-  </svg>
-
+  <span shrink-0 aspect="1/1" sm:h-8 xl:h-10 class="rtl-flip">
+    <svg px-5 block sm:hidden xl:block width="140" height="40" viewBox="0 0 140 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (jpezninjo) [inline components/nav/NavLogo.vue:3]: I adapted the `block sm:hidden xl:block` here from the disappearing Menu item texts

**Review by wtfluckey — APPROVED** [review]
Great work! Love the idea of swapping out the desktop/mobile assets at the svg level instead of having two separate svg files and doing at the template level.

**Comment by jpezninjo** [comment]
@wtfluckey thank you! Using 1 asset just made sense to me when the Elk logo already had the code in 😄

**Pull request merged**